### ISSUE TITLE
Suggestion: Alternative Project View which shows all projects and branches

### ISSUE THREAD

**Issue by dougthor42** [appveyor/website#84]
First, let me say that I suggested this exact same thing to Travis-CI [here](https://github.com/travis-ci/travis-web/issues/380), and I'm copying most of the post, really just changing an image or URLs.

I find that the user Project View ([example](https://ci.appveyor.com/projects)) lacks information.

![image](https://cloud.githubusercontent.com/assets/5386897/8861907/f0570a5e-3141-11e5-95ed-6649f342a856.png)

Specifically, a user cannot see all branches of all projects at once.

I propose adding an alternate view which fixes this. I've included a (very crude) mockup - a simple grid view where columns are various projects, and rows are their branches:

![image](https://cloud.githubusercontent.com/assets/5386897/8861578/09066948-3140-11e5-9122-1a431cd3c63e.png)

As you can see, all the same information is still there (Build #, Commit #, duration, end_time, status via color), but now we can see all branches. It would also be nice if the user could transpose the matrix, for those times when users have many more projects than they do branches.

Another example would be [BuildBot](http://buildbot.net/)'s [grid display](http://buildbot.wxpython.org:8010/grid), however this is only for one project at a time.

I was thinking of trying to do this myself within a standard webpage, but I've so far been unable to find any API that I could use to grab a user's (my) projects and branches.

I'll let your designers add all the bells and whistles (icons, proper font/color, etc.), but I think I'm getting my idea across.

Thanks,

**Comment by FeodorFitsner** [comment]
Well, building dashboard around branches does makes sense, kind of "git-flow"-oriented design. Maybe this could be presented in a different way, like:

```
- Project 1
     branch-A
     branch-B
- Project 2
     branch-A
     branch-B
     branch-C
+ Project 3
+ Project 4
```

**Comment by dougthor42** [comment]
That is definitely another good option. It removes the empty space that my design would have for projects that have few or no branches.

Personally, I'd like all project trees to be expanded by default, but I'm sure some other users wouldn't. Perhaps if it just remembered the most recent expand/collapse state?

**Comment by FeodorFitsner** [comment]
Yeah, another consideration is performance as there is nore information to retrive and display on a screen.
